_posts/en/newsletters/2020-03-04-newsletter.md

```diff
@@ -0,0 +1,226 @@
+---
+title: 'Bitcoin Optech Newsletter #87'
+permalink: /en/newsletters/2020/03/04/
+name: 2020-03-04-newsletter
+slug: 2020-03-04-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes a proposed update to BIP340 schnorr
+keys and signatures, seeks feedback on a proposal to improve startup
+feature negotiation between full nodes, examines a suggestion for a
+standardized way to prevent hardware wallets from using corrupt nonces
+to leak private keys, links to an analysis of the properties necessary
+in a hash function for taproot to be secure, and summarizes recent
+topics from the Bitcoin Core PR Review Club.  Also included are our
+regular sections for release announcements and notable changes to
+popular Bitcoin infrastructure projects.
+
+## Action items
+
+*None this week.*
+
+## News
+
+- **Updates to BIP340 schnorr keys and signatures:** as previously
+  mentioned in two separate items in Newsletter #84 ([1][news83 safety], [2][news83 tiebreaker]),
+  several updates to [BIP340][] schnorr signatures have been proposed
+  (which also affect [BIP341][] taproot).  Pieter Wuille [suggests][wuille
+  update] changing which variant of a public key should be used with
+  [BIP340][], with the new selection being based on the eveness of the
+  key.  Also, the recommended procedure for generating a nonce has been
+  changed to include the public key in nonce generation, to include
+  extra randomness when available, and to include a step in the nonce
```

Comment:
perhaps "in order to increase randomness" to not repeat "to include" in the same sentence